plugins/http2_adapter/lib/src/client_setting.dart

```diff
@@ -13,4 +13,6 @@ class ClientSetting {
   /// the connection or not.  The handler should return true
   /// to continue the [SecureSocket] connection.
   bool Function(X509Certificate certificate)? onBadCertificate;
+
+  Uri? proxy;
```

Comment:
Please add some documentation here, because I'm kinda lost what to do with it.

Reply:
@ueman added